applications/nrf_desktop/configuration/nrf52840dongle_nrf52840/prj_release.conf

```diff
@@ -47,6 +47,7 @@ CONFIG_DESKTOP_WATCHDOG_ENABLE=y
 CONFIG_DESKTOP_FAILSAFE_ENABLE=y
 
 CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
```

Comment:
Should we use B0 consistently also for prj_3bleconn and prj_4llpmconn ?

Reply:
Lets do it in another task